### ISSUE TITLE
📎 Biome should report the syntax error when trying formatting an invalid code 

### ISSUE THREAD

**Issue by Conaclos** [biomejs/biome#1626]
### Description

Biome is able to format malformed code. However, when I am trying to format a malformed code, Biome errors telling me that `format with errors is disabled`. This is not helpful. Biome should suggest an option to turn on formatting with error. It should also inform me the syntax error that it encountered.

```sh
❯ cat index.js 
let val = ;

❯ npx biome format index.js 
index.js format ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Format with errors is disabled.

Compared 1 file(s) in 676µs
Skipped 1 file(s)
internalError/io ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ No files were processed in the specified paths.
```

**Comment by logankaser** [comment]
If it fails to format at all, it the process should exit with a non-zero status as well.
Currently, it fails and displays this error, but returns `EXIT_SUCCESS`
